tests/test_sklearnex.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+import numpy as np
+
+from sklearn.cluster import DBSCAN
+
+class TestSklearnex(unittest.TestCase):
+    def test_dbscan(self):
+        from sklearnex import patch_sklearn
+        patch_sklearn()
```

Comment:
```suggestion

class TestSklearnex(unittest.TestCase):
    def test_dbscan(self):
        from sklearnex.cluster import DBSCAN
```
I think that for package validation perspective - this would be the simplest way

There are 2 things related to patching:
1. You have to reimport "from sklearn.cluster import" after patching. Because patching process wouldn't affect already existing instances.
2. Patch would impact entire process - i'm not 100% sure how this would behave in case of the tests, but i'm afraid that it would have effect on following scikit calls, so unpatch would be required.

Reply:
Great suggestion. Applied. Thank you :)